benchmark/cluster_containers_setup.py

```diff
@@ -153,13 +181,20 @@ def __init__(self, reactor, env, options):
         """
         self.options = options
         try:
-            self.application_template = self.options['template']
+            self.image = DockerImage(repository=self.options['image'])
+            self.max_size = int(GiB(self.options['max-size']).to_Byte().value)
+            self.mountpoint = unicode(self.options['mountpoint'])
             self.per_node = self.options['apps-per-node']
             self.control_node_address = self.options['control-node']
             self.timeout = self.options['wait']
+            self.wait_interval = self.options['wait-interval']
             if self.timeout is None:
                 # Wait two hours by default
                 self.timeout = 72000
+            self._num_loops = 10
+            if self.wait_interval < self.timeout:
```

Comment:
By default they'll be equal so by default there will be ten loops? Maybe default should be 1, not 10.